HW04/tim_bulat/spec/mentor_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require_relative '../lib/student'
+require_relative '../lib/mentor'
+require_relative '../lib/homework'
+require_relative '../lib/notification'
+
+RSpec.describe Mentor do
+  let(:student) { Student.new(name: 'students_name', surname: 'students_surname') }
+  let(:mentor) { described_class.new(name: 'mentor_name', surname: 'mentor_surname') }
+  let(:hw) { Homework.new(title: 'title', description: 'description', student: student, mentor: mentor) }
+
+  context 'when created' do
+    it 'returns correct name' do
+      expect(mentor.name).to eq 'mentor_name'
+    end
+
+    it 'returns correct surname' do
+      expect(mentor.surname).to eq 'mentor_surname'
+    end
+  end
+
+  context 'when subscribed to student' do
+    it 'student added' do
```

Comment:
you should describe in `it block` what the method does. the line 31 is a good example